girder/api/rest.py

```diff
@@ -1198,6 +1198,9 @@ def sendAuthTokenCookie(self, user=None, scope=None, token=None, days=None):
         if cherrypy.request.scheme == 'https' or cherrypy.request.base.startswith('https'):
             cookie['girderToken']['secure'] = True
 
+        if domain := Setting().get(SettingKey.COOKIE_DOMAIN):
```

Comment:
I think this is why the integration test is failing; this language feature isn't supported on our minimum supported python version (sadly).

Reply:
`setup.py` has `python_requires='>=3.8'` that is 3.8 feature

Reply:
`'bad'` was passing the validator for COOKIE_DOMAIN. I fixed it.